hudi-client/hudi-java-client/src/test/java/org/apache/hudi/common/table/read/HoodieFileGroupReaderOnJavaTestBase.java

```diff
@@ -113,4 +132,65 @@ public void commitToTable(List<HoodieRecord> recordList, String operation, boole
       }
     }
   }
+
+  @Override
+  public void commitSchemaToTable(InternalSchema schema, Map<String, String> writeConfigs, String historySchemaStr) {
```

Comment:
Is there any reason we need each engine to implement this? Could we have each implementation just rely on the java writer?